hardware/deconz/Deconz.cpp

```diff
@@ -0,0 +1,824 @@
+#include "stdafx.h"
+#include <string>
+#include "Deconz.h"
+#include "../../main/Logger.h"
+#include "../../main/EventsPythonDevice.h"
+#include "../../main/localtime_r.h"
+#include "../../json/reader.h"
+#include "../../main/WebServer.h"
+#include <platform/Log.h>
+#include "../../main/mainworker.h"
+#include "../../main/SQLHelper.h"
+
```

Comment:
Could you add a link to the github page somewhere at the top here?
Maybe with some more information (zigbee, supported hardware etc)

Otherwise we do not know what this kind of hardware is